api/agent.go

```diff
@@ -96,6 +96,15 @@ type AgentToken struct {
 	Token string
 }
 
+// Metrics info is used to store different types of metric values from the agent.
+type MetricsInfo struct {
+	Timestamp string
+	Gauges    []map[string]interface{}
```

Comment:
These would be more useful with concrete types. The structs can be defined in the API package.